libs/expo/betterangels/src/lib/ui-components/RequestedProvidedServices/ServicesModal.tsx

```diff
@@ -195,46 +196,31 @@ export default function ServicesModal(props: IServicesModalProps) {
   };
 
   const closeModal = () => {
-    const newInitialServices = initialServices
-      // TODO: remove after cutover
-      .filter((item) => !!item.serviceEnum)
-      .filter((item) => item.serviceEnum !== ServiceEnum.Other)
-      .map((service) => ({ id: service.id, enum: service.serviceEnum! }));
-    const initialServiceOthers = initialServices
-      .filter((item) => item.serviceEnum === ServiceEnum.Other)
-      .map((service) => ({
-        id: service.id,
-        title: service.serviceOther || null,
+    const existing: SelectedService[] = initialServices
```

Comment:
please rename to `initialServiceRequests`. There are actually a lot of places where "service requests" are referred to as "services", and we should probably update all of those var names. It will be more difficult to read and understand the frontend, if we don't.